yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -80,11 +80,7 @@ var ybValueConverterSuite = map[string]ConverterFn{
 		}
 		epochSeconds := epochMicroSecs / 1000000
 		epochNanos := (epochMicroSecs % 1000000) * 1000
-		microTimeStamp, err := time.Parse(time.RFC3339Nano, time.Unix(epochSeconds, epochNanos).Local().Format(time.RFC3339Nano)) //TODO: check if proper format for Micro can work
-		if err != nil {
-			return columnValue, err
-		}
-		timestamp := strings.TrimSuffix(microTimeStamp.String(), " +0000 UTC")
```

Comment:
Is trimming the suffix not required anymore with `UTC()`?

Reply:
Ah, no. 
trimming has become unnecessary because now I've got rid of RFC3339Nano and replaced it with a custom format - for micro and nano (i just removed the timezone part of the RFC3339Nano so now we don't have to remove the UTC from the string)

Reply:
Oh okay. Yes. Thanks!